curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-fetch-method.md

```diff
@@ -0,0 +1,165 @@
+---
+id: 5ccfad82bb2dc6c965a848e5
+title: Get JSON with the JavaScript fetch method
+challengeType: 6
+---
+
+## Description
+<section id='description'>
+Another way to request external data is to use the JavaScript <code>fetch()</code>  method. It is equivalent to XMLHttpRequest, but the syntax is considered easier to understand.
+Here is the code for making a GET request to <code>/json/cats.json</code>
+
+```js
+
+fetch('/json/cats.json')
+	.then(response => response.json())
+	.then(data => {
+		document.getElementById('message').innerHTML=JSON.stringify(data);
+	})
+
+```
+Let's analyze each piece of this code. 
```

Comment:
```suggestion

Take a look at each piece of this code. 
```
The space is because we're in the process of adding a linter (and this would fail).  The rewording is to conform with the [style guide](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-curriculum-challenges.md#writing-instructions): 
> Challenge text should use the second person ("you") to help to give it a conversational tone. This way, the text and instructions seem to speak directly to the camper working through the challenge. Try to avoid using the first person ("I", "we", "let's", and "us").